lighthouse-core/runner.js

```diff
@@ -321,9 +321,7 @@ class Runner {
 
       Sentry.captureException(err, {tags: {audit: audit.meta.id}, level: 'error'});
       // Errors become error audit result.
-      const errorMessage = err.friendlyMessage ?
-        `${err.friendlyMessage} (${err.message})` :
-        `Audit error: ${err.message}`;
+      const errorMessage = err.friendlyMessage ? err.friendlyMessage : err.message;
```

Comment:
so what errors do we start losing for the code for after this, just the speedline ones? or the page load ones too?

I agree there's not a strong need for our users to know the code, but having it somewhere in the LHR would be nice.

Reply:
I think we should follow the same pattern of splitting these out into unique values.